src/themes/teams/components/Avatar/avatarStyles.ts

```diff
@@ -3,27 +3,33 @@ import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/t
 import { IAvatarPropsWithDefaults } from '../../../../components/Avatar/Avatar'
 
 const avatarStyles: IComponentPartStylesInput<IAvatarPropsWithDefaults, any> = {
-  root: ({ props: { size } }): ICSSInJSStyle => ({
-    position: 'relative',
-    backgroundColor: 'inherit',
-    display: 'inline-block',
-    verticalAlign: 'middle',
-    height: pxToRem(size),
-    width: pxToRem(size),
-  }),
+  root: ({ props: { size } }): ICSSInJSStyle => {
+    const remSize = pxToRem(size)
```

Comment:
probably, it would be better to name it as `sizeInRem` - otherwise it might mislead by providing notion of `how many pixels are in rem`

Reply:
done